elements/lisk-transactions/test/utils/state_store_mock.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { Account, TransactionJSON } from '../../src/transaction_types';
+import {
+	StateStoreCache,
+	StateStoreDefaultGetter,
+	StateStoreGetter,
+	StateStoreSetter,
+	StateStoreTransactionGetter,
+} from '../../src/base_transaction';
+
+export const defaultAccount = {
```

Comment:
for the Account class,  i think it's better to use the real `Account` so that it won't have inconsistency in the future

Reply:
What you meant by real account?

Reply:
ah nevermind, i was thinking it's `lisk-chain`, but it's transactions
All good here